docs/docs/learning.md

```diff
@@ -5,5 +5,84 @@ sidebar_label: Learning Resources
 slug: /learning
 ---
 
-TODO
+## **What is SQL?**
 
+SQL is often pronounced "sequel," but it stands for "structured query language." You can think of it as the computer equivalent of asking verbal questions.
+
+Let's say you're interested in the Calls for Service (911 calls) dataset here on nolabase. You might want to know, "How many robberies have there been recently?” or “What's the most common emergency people call in for?"
+
+If you were on the phone with someone, you would ask those questions verbally. On a computer, you would ask them using a query language like SQL. Luckily, there is a lot of overlap between the human and computer versions of these questions.
+
+Typically, SQL is reserved for datasets that are too big to fit in spreadsheet programs like Google Sheets or Excel. It's popular among businesses, schools, software engineers, government agencies and others who need to present a large amount of information in a more manageable way.
+
+If this is your first time using SQL, you can use this guide to write your first queries and get a sense of what you can do.
+
+
+## **Getting ready**
+
+To use SQL on datasets within nolabase, you need to connect with Excel, Tableau, or another method. If you're unsure how to do this, there are instructions on the nolabase site under the "Connecting" section.
+
+With any dataset, the first thing you should do is get your bearings. On nolabase, the Overview page for each dataset shows the data that is available as well as important things to know. For instance, the Calls for Service (911 calls) data omits calls regarding juveniles. That could have a big effect on the outcome of your analysis, depending on what you're looking at.
+
+There are many things that can go wrong with data - misspellings, empty rows, a strange column name you don't understand. To get an idea, you can use the [Quartz Guide to Bad Data](https://github.com/Quartz/bad-data-guide) to see dozens of the most common problems that crop up.
+
+If you have any questions or are unsure, the most foolproof method is to contact the source of the data itself. With the Calls for Service dataset, that is the New Orleans Police Department. Many, if not most, sources will have a phone number or email you can contact with general questions.
+
+
+## **SQL Queries**
+
+To use SQL, you'll need to understand a few basic concepts and commands. The language uses a certain syntax, or word order, to converse with a database. The key word here is in the middle name of the language: query. In this case, a query is a question written in SQL syntax.
+
+For example, this query asks, "Who has permits for short term rentals (AirBnBs) in New Orleans?"
+
+```
+SELECT contact_name
+FROM str_permits
+WHERE current_status = issued
+```
+
+"SELECT" tells the computer what data you want to see. In this case, we're asking to see the licensees' names. If you want to see all the columns at once, you can type an asterisk "*" to mean "all."
```

Comment:
You can use the single back tick to set apart code keywords in monospace typeface, something like this:
```suggestion
`SELECT` tells the computer what data you want to see. In this case, we're asking to see the licensees' names. If you want to see all the columns at once, you can type an asterisk `*` to mean "all."
```
Similar for the ones below too.